vtStorAta/DriveEnumeratorAta.cpp

```diff
@@ -32,16 +32,13 @@ cDriveEnumeratorAta::~cDriveEnumeratorAta()
 
 }
 
-eErrorCode cDriveEnumeratorAta::EnumerateDrives( Vector_Drives& AddToList, U32& Count )
+eErrorCode cDriveEnumeratorAta::EnumerateDrives(std::vector<String> PathsList, Vector_Drives& AddToList, U32& Count )
```

Comment:
This is passed by value, that means a new vector is create and everything copied over.